docs/src/pages/[platform]/connected-components/storage/storageimage/examples/DefaultStorageImageExample.tsx

```diff
@@ -1,5 +1,5 @@
 import { StorageImage } from '@aws-amplify/ui-react-storage';
 
 export const DefaultStorageImageExample = () => {
-  return <StorageImage alt="cat" imgKey="cat.jpg" accessLevel="guest" />;
+  return <StorageImage alt="sleepy-cat" path="guest/cat.jpg" />;
```

Comment:
Just remembered that the `guest` access level folder is actually located at `public`
```suggestion
  return <StorageImage alt="sleepy-cat" path="guest/cat.jpg" />;
```
See: https://github.com/aws-amplify/docs/pull/7323/files#diff-7fb300718c1f67687e40188fc32c5b58e1fb909ddf913a0cfcaaf4c8a67cdd37R129

Reply:
Oh good catch